### ISSUE TITLE
"Open Project" button doesn't do anything

### ISSUE THREAD

**Issue by audiodude** [nitrous-io/nitrogen-dist#12]
I have already clicked the "File Sync" button, and it appears to be green and sync'd.

However clicking "Open Project" does nothing.

This is on OS X. Here is the log from the toolbar Nitrogen menu.

```
Running Nsync...
[nsync event] {"name":"pid","data":35323}
[nsync event] {"name":"listening","data":64876}
[nsync event] {"name":"container.sync-enabled","data":{"slug":"audiodude-103220"}}
[nsync event] {"name":"container.sync-started","data":{"slug":"audiodude-103220"}}
[nsync event] {"name":"container.sync-finished","data":{"slug":"audiodude-103220"}}
```

**Comment by styner32** [comment]
It does not show anything on the file tree? "Open Project" is to show current container folder in the file tree. It will not do anything if the folder is already there.

**Comment by PudparK** [comment]
I'm having the same problem as @styner32 .

**Comment by styner32** [comment]
@PudparK Do you also have the same problem as @audiodude ? Could you try to open project with atom by running `atom $HOME/Nitrous/<host-slug>/<container-slug>` and see if it works?

**Comment by audiodude** [comment]
I tried it again just now, and "Open Project" now opens a pane on the left side with a list of files.

**Comment by PudparK** [comment]
@styner32 where would I run this? That's just a little over my head.

**Comment by styner32** [comment]
@PudparK Ah Sorry. What OS do you use? If it is MacOS, you can run it in the terminal.
I just wanted to see if the container folders are created correctly.
Or could you check if the folders are created under `$HOME/Nitrous`?

**Comment by PudparK** [comment]
@styner32 pc... don't judge me.

**Comment by PudparK** [comment]
Run it in PowerShell?

**Comment by styner32** [comment]
Yes. `atom $HOME/Nitrous` in power shell. and could you check if all files are synced properly?

**Comment by PudparK** [comment]
@styner32 that brought up the Nitrous dir / username-### dir / project_name ###### dir. But it did not bring up what's inside the project directory. Does this make sense?

**Comment by styner32** [comment]
Ah that seems files are not successfully synced or no files in your container. Do you have files in `/home/nitrous/code` folder in your container? It only sync files under the folder.

Could you also reply me via `Send feedback` in atom?

![screen shot 2016-02-05 at 12 29 58 pm](https://cloud.githubusercontent.com/assets/302481/12838090/349040ca-cc04-11e5-9f73-714988632a8f.png)

So we can share more details in your container.

**Comment by PudparK** [comment]
@styner32 Responded just now.

**Comment by nsymtex** [comment]
Same problem. Clicking "Open Project" does nothing. Files did appear to be synced. 

C:\Users\User\Nitrous\user-#### does have files in it ~90MB

If I force tree view open with ctrl+\ there is nothing there except for the container name

Windows 7

**Comment by styner32** [comment]
Hi. I just found that "Open Project" does not show anything if tree-view is not enabled or tree-view is hidden.
We will fix the issue.

@nsymtex Do you mean it shows empty folder?

**Comment by nsymtex** [comment]
That is correct, styner32

Tree view does not show files below parent folder no matter what I do, pressing Open Project does not appear to have any function with our without tree view forced open

I will test updates

Sorry if English is bad

**Comment by styner32** [comment]
@nsymtex Can you also reply me via Send feedback in atom? So we can share more details?
